commit 634d4e35b4bfd7e3df18c12e863463970884682b
Author: Lamarque V. Souza <lamarque@gmail.com>
Date:   Mon Aug 22 21:35:21 2011 -0300

    Change the default ssid name for shared connections to match the
    connection name.

diff --git a/libs/ui/wirelesspreferences.cpp b/libs/ui/wirelesspreferences.cpp
index 304b053..1288728 100644
--- a/libs/ui/wirelesspreferences.cpp
+++ b/libs/ui/wirelesspreferences.cpp
@@ -69,7 +69,7 @@ WirelessPreferences::WirelessPreferences(bool setDefaults, const QVariantList &a
         if (args.count() > 3 && args[3].toString() == QLatin1String("shared")) {
             static_cast<Knm::Ipv4Setting *>(m_connection->setting(Knm::Setting::Ipv4))->setMethod(Knm::Ipv4Setting::EnumMethod::Shared);
             shared = true;
-            ssid = i18n("Shared_Wifi");
+            ssid = i18n("Shared_Wireless_Connection");
         }
     } else {
         kWarning() << "Could not find deviceUni or AP UNI in args:" << args;
